tests/containers/cc/Makefile

```diff
@@ -35,10 +36,22 @@ run: run-hw run-sw
 
 ifeq (${SGXLKL_PREFIX},)
 run-hw:
-	@echo SGXLKL_PREFIX missing, skipping test
+	@echo SGXLKL_PREFIX missing, skipping test.
+	@echo For local testing, run the following first:
+	@echo "> cd $$SGXLKL_ROOT"
+	@echo "> export SGXLKL_PREFIX=$$SGXLKL_ROOT/build/install"
+	@echo "> make install PREFIX=\$$SGXLKL_PREFIX"
```

Comment:
So if I understand this correctly, you can't test without doing an install? Is there a reason it's different than the others where you can test without doing install?

Reply:
That's correct. It is different because all other tests run SGX-LKL on the host, whereas this test runs SGX-LKL inside a Docker container where the SGX-LKL distribution/installation is mounted into it. Mounting the build folder is not possible.

Reply:
I think that difference is worth the inclusion of a README that notes why it's different. A less knowledgeable person, such as myself might think that was unintentional.

Otherwise, this looks good.

Reply:
I moved it into a README. Could you have another look?